docs/main/reference/support-policy.mdx

```diff
@@ -20,13 +20,14 @@ Given the reality of time and resource constraints as well as the need to suppor
 
 The current status of each Capacitor version is:
 
-| Version |      Status      |   Released    | Maintenance Ends  | Ext. Support Ends |
-| :-----: | :--------------: | :-----------: | :---------------: | :---------------: |
-|   V5    |    **Active**    |  May 3, 2023  |        TBD        |        TBD        |
-|   V4    | Extended Support | July 27, 2022 | November 3, 2023  |    May 3, 2024    |
-|   V3    |  End of Support  | May 18, 2021  | January 27, 2023  |   July 27, 2023   |
-|   V2    |  End of Support  | April 3, 2020 | November 18, 2021 |   May 18, 2022    |
-|   V1    |  End of Support  | May 23, 2019  |  October 3, 2020  |   April 3, 2021   |
+| Version |      Status      |    Released    | Maintenance Ends  | Ext. Support Ends |
+| :-----: | :--------------: | :------------: | :---------------: | :---------------: |
+|   V6    |    **Active**    | April 15, 2024 |        TBD        |        TBD        |
+|   V5    |   Maintenance    |   May 3, 2023  | October 15, 2024  |  April 25, 2025   |
```

Comment:
```suggestion
|   V5    |   Maintenance    |   May 3, 2023  | October 15, 2024  |  April 15, 2025   |
```
Shouldn't this be 15?

Reply:
ah yep... fat fingered that. Thanks for catching it!